util/sync_point.cc

```diff
@@ -35,8 +36,13 @@ void TestKillRandom(std::string kill_point, int odds,
 }
 
 SyncPoint* SyncPoint::GetInstance() {
-  static SyncPoint sync_point;
-  return &sync_point;
+  static std::once_flag flag;
```

Comment:
While this change on its own is fine I just want you to be aware of two things:

- MSVC15 properly implements C++11 in this regard so statics creation becomes thread-safe and no change in necessary then.

- SyncPoint is a test ONLY facility and its code is not a part of the final product. Just want to make sure you do not base any serious production features on top of it such as "rapid creation of many databases".